python/ray/data/grouped_dataset.py

```diff
@@ -279,6 +293,10 @@ def max(self, on: Optional[AggregateOnTs] = None) -> Dataset[U]:
                 - For an Arrow dataset: it can be a column name or a list
                   thereof, and the default is to do a column-wise max of all
                   columns.
+            ignore_nulls: Whether to ignore null values. If ``True``, null
```

Comment:
Can we define what the null values are since it's not as simple as None, maybe just referencing pandas null?